### DISCUSSION TITLE
Using Arisan with a siemens Simatic P7-300 (on a probat P75)

### DISCUSSION THREAD

**Discussion by JwfGroenen** [artisan-roaster-scope/artisan#1265] [Versions]
I am looking for a download of the P7-300 driver to be imported into the Artisan software. Can anybody help me with that? Thanks

**Comment by poundy** [comment]
I don't know what a P7-300 is, do you mean S7-300?  Artisan doesn't need drivers, your OS does - you'd be best approaching Probat to seek out drivers, if you need them.  From the Artisan manual page, the supported connection is Modbus TCP, so a network not USB based connection, so I suspect you don't need a driver.... https://artisan-scope.org/machines/probat/

**Comment by MAKOMO** [comment]
Those older Probat P75 are not (yet) supported. The PLC seems to be a Simatic S7-300 which should be able to communicate via the S7 protocol, which is supported by Artisan. However, we don't have any information on the register assignment Probat implemented in its firmware to come up with the correct Artisan configuration. We would need that technical information on this from Probat to add support.
